### PR TITLE
fix(theming): logo not visible on Windows 11 light mode

### PR DESCRIPTION
**Pull request by edgy-b**
This PR adds a dark logo for light mode and makes sure the logo changes depending on the user's theme.

### CODE REVIEW

**Comment by AnonymousWP** [comment]
Converted this one to a draft because we're still testing.

**Comment by AnonymousWP** [comment]
It's now confirmed to work on Windows 10, but not 11 yet for me.

**Comment by AnonymousWP** [comment]
It's an issue on my end, works elsewhere. Marking this PR as ready. @Universal-Debloater-Alliance/developers can review this now.

**Comment by Frigyes06** [comment]
Changing the logo based on system theme and not application setting feels odd to me. We should synchronize the whole app theme with system theme at first startup, then let the user overwrite it in the settings. Or have a setting to synchronize only the logo. We need to allow for more control over this setting.

File: resources/assets/logo-dark.png
Comment (Frigyes06) [inline resources/assets/logo-dark.png:1]: naming convention, rename this to logo-light, the other one to logo-dark

**Review by Frigyes06 — CHANGES_REQUESTED** [review]

**Review by Frigyes06 — APPROVED** [review]

**Pull request merged**